control-plane/csi-driver/src/bin/node/mount_utils/bin/mount_utils.rs

```diff
@@ -0,0 +1,50 @@
+use crate::error::MountError;
+
+use sys_mount::{FilesystemType, Mount, MountFlags, UnmountFlags};
+
+/// Calls sys mount's mount in a spawn blocking.
+pub(crate) async fn mount(
```

Comment:
If the mount/unmount calls get stuck due to the device being in a bad state, how are you ensuring you don't keep stacking up new processes trying to the same operation?

Reply:
In my testing I did not see any process stacking up.

Reply:
When the problem happens(repro), have you checked the host's dmesg if it shows any softlockup or oops message from kernel? In that case the host is already unreliable.

Reply:
No, I haven't seen that, also seems like the process is not blocking the CPU core in which case the softlockup shows up probably, I can see other tasks getting scheduled on the core where the ioctl process is running.